OloEngine/src/OloEngine/Core/Hash.cpp

```
@@ -72,4 +72,22 @@ namespace OloEngine
         return CRC32(string.c_str());
     }
 
+    u32 Hash::CRC32(const void* data, sizet size)
+    {
+        OLO_PROFILE_FUNCTION();
+
+        if (!data || size == 0)
+        {
+            return 0;
+        }
```

Cppcheck (2.20.0)

[style] The function 'Hash64' is never used.

(unusedFunction)